### PR TITLE
docs: note difference between solutions and github install

### PR DESCRIPTION
**Pull request by Bingjiling**
Issue #, if available:

Description of changes:

* Doc update related to https://github.com/awslabs/fhir-works-on-aws-deployment/issues/408 

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ FHIR Works on AWS utilizes AWS Lambda, Amazon DynamoDB, Amazon S3 and Amazon Ela
 
 The easiest and quickest way to access FHIR Works on AWS is by using [AWS solution](https://aws.amazon.com/solutions/implementations/fhir-works-on-aws/). To modify the code and set up a developer environment, follow the steps below:
 
+**Note**: AWS Solution provides an earlier version of FWoA install, please follow the instruction below to install from GitHub repository if you wish to install the latest version.
```
Comment (carvantes) [inline README.md]: Can you add a link to the AWS solution here?

I think that we don't mention the solution anywhere else and without a link most customers won't understand this note

**Review by carvantes — APPROVED** [review]
LGTM, just a minor comment

File: README.md (hunk 1 above)
Comment (Bingjiling) [reply README.md]: Sure, I'll add a link to our solutions branch

Comment (carvantes) [reply README.md]: I was thinking of a link to the Soultion page: https://aws.amazon.com/solutions/implementations/fhir-works-on-aws/?did=sl_card&trk=sl_card

Comment (Bingjiling) [reply README.md]: The Solutions link is present on line 18

**Pull request merged**